languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,40 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="ACCORD_TOUT_ADJECTIF" name="accord tout adjectif" default="temp_off">
+          <rule>
+              <antipattern>
+                  <token>tout</token>
+                  <token regexp="yes">publique?|public?</token>
+              </antipattern>
+              <pattern>
+                  <token postag="(P\+)?D.*" postag_regexp="yes"/>
+                  <token postag="N f s" postag_regexp="yes"/>
+                  <marker>
+                      <token>tout</token>
+                  </marker>
+                  <token postag="J f s" postag_regexp="yes"><exception regexp="yes">^[aâeéèêioôœuhy].*|&H_ASPIRE;</exception></token>
```

Comment:
Okay, so here's an example of what I wanted you to do here. Could you please check it for false positives?

```xml
          <rule>
              <antipattern>
                  <token>tout</token>
                  <token regexp="yes">publique?|public?</token>
              </antipattern>
              <pattern>
                  <token postag="(P\+)?D.*" postag_regexp="yes"/>
                  <token postag="N f s" postag_regexp="yes"/>
                  <marker>
                      <token>tout</token>
                  </marker>
                  <token postag="J [ef] sp?" postag_regexp="yes">
                    <exception regexp="yes">^[aâeéèêioôœuhy].*|&H_ASPIRE;</exception>
                    <exception regexp="yes">membre|rebelle|juste</exception>    
                </token>
              </pattern>
              <message>Vérifiez l’accord entre « \1 » et « \3 ».</message>
              <suggestion>toute</suggestion>
              <example correction="toute">Une veste <marker>tout</marker> neuve.</example>
              <example>Une veste <marker>tout</marker> publique.</example>
          </rule>
```

Btw, you the rule editor doesn't know the entity `&H_ASPIRE;` – in this case, you can ignore it and just ignore any matches that occur to a not recognized "h aspiré". Please give me feedback on that here as a comment.

Reply:
I didn't see any &H_ASPIRE; in the rule editor so I think that's okay.

Reply:
I also checked for FPs and I extended the exceptions.